features/events/syslog/src/main/java/org/opennms/netmgt/syslogd/Rfc5424SyslogParser.java

```diff
@@ -114,18 +114,20 @@ protected SyslogMessage parse() throws SyslogParserException {
         return message;
     }
 
-    protected static Date parseDate(final String dateString) {
+    protected Date parseDate(final String dateString) {
         if (dateString.endsWith("Z")) {
             try {
                 final DateFormat df = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss'Z'", Locale.ROOT);
                 df.setTimeZone(TimeZone.getTimeZone("GMT"));
```

Comment:
@j-white I see here an inconsistent behavior between Rfc5424SyslogParser.java (defaults to GMT if no default time zone is given) and SyslogParser.java (defaults to TimeZone.getDefault() == the time zone of the JVM). Should we leave this or also change Rfc5424SyslogParser.java to use TimeZone.getDefault()?

Reply:
@patrick-schweizer Good catch. I would opt to change the default behavior of the `Rfc5424SyslogParser` as well to keep it consistent among parsers. Would like to hear what others think though.

Reply:
I am usually for a consistent behaviour, unless it is not possible and then document why that is. However which implication a change of `Rfc5424SyslogParser` has, I cannot say.